src/components/10-atoms/input-file/index.react.d.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Icon } from '@axa-ch/icon/lib/index.d';
+
+type Variant = 'secondary' | 'red' | 'inverted';
+
+interface AXAButtonProps {
```

Comment:
I guess that's a copy pasta issue. Better chose `AXAInputFileProps` here.

Reply:
@NoahLiechti yes please change that

Reply:
I will. Thanks for the feedback!